### PR TITLE
Add hostname radical to the ZIM tags #147

### PR DESCRIPTION
**Pull request by kelson42**

### CODE REVIEW

File: lib/Zim.js [hunk 1]
```diff
@@ -99,6 +100,24 @@ Zim.prototype.computeZimName = function() {
...
+    var tags = [];
+
+    /* Mediawiki hostname radical */
+    var mwUrlHostParts = urlParser.parse(this.env.mw.base).host.split(".");
+    var mwUrlHostPartsTag = mwUrlHostParts.length > 1 ?
+	mwUrlHostParts[mwUrlHostParts.length - 2] : mwUrlHostParts[mwUrlHostParts.length - 1]
+    tags.push(mwUrlHostPartsTag.toLowerCase());
+
+    /* novid/nopic */
+    this.env.nopic ? tags.push('nopic') : this.env.novid ? tags.push('novid') : undefined;
```
Comment (subbuss) [inline lib/Zim.js]: I don't like using ternary operators for this ... Better to use the same if (this.env...) pattern you use below on lines 115 and 116.

File: lib/Zim.js [hunk 2]
```diff
@@ -99,6 +100,24 @@ Zim.prototype.computeZimName = function() {
     return (this.publisher ? this.publisher.toLowerCase() + '.' : '') + this.env.computeFilenameRadical(false, true, true);
 };
 
+Zim.prototype.computeZimTags = function() {
+    var tags = [];
+
+    /* Mediawiki hostname radical */
+    var mwUrlHostParts = urlParser.parse(this.env.mw.base).host.split(".");
+    var mwUrlHostPartsTag = mwUrlHostParts.length > 1 ?
+	mwUrlHostParts[mwUrlHostParts.length - 2] : mwUrlHostParts[mwUrlHostParts.length - 1]
```
Comment (subbuss) [inline lib/Zim.js]: I don't have an opinion on this .. depends on what the requirement are for tags.

File: lib/Zim.js [hunk 3]
```diff
@@ -99,6 +100,24 @@ Zim.prototype.computeZimName = function() {
...
+    tags.push(mwUrlHostPartsTag.toLowerCase());
```
Comment (subbuss) [inline lib/Zim.js]: https://github.com/openzim/mwoffliner/issues/147#issuecomment-350498365 wanted first letter to be uppercased ... not use how PhET will be generated however.

File: lib/Zim.js (hunk 1 above)
Comment (kelson42) [reply lib/Zim.js]: Agree

File: lib/Zim.js (hunk 2 above)
Comment (kelson42) [reply lib/Zim.js]: Kind of automatic solution to "categorize" somehow ZIM files.

File: lib/Zim.js (hunk 3 above)
Comment (kelson42) [reply lib/Zim.js]: Yes, I do not see how to do that everywhere, but I have created a new --customZimTags option which goes in that direction if we really want to do that in the future.

**Pull request merged**